README.md

```diff
@@ -58,13 +47,16 @@ I tried to make the Vagrantfile as readable as possible but it's worth repeating
 |---|---|---|---|
 |8800|API|oms-core|8080|
 |8801|consumer|oms-profiles-module|8081|
+|---|API|oms-events|8082|
```

Comment:
have a reverse-proxy on :8803/api =)
In the end we should set up one with nginx anyway, so I did it already.